frb_codegen/src/main.rs

```diff
@@ -1,21 +1,46 @@
-use std::process::exit;
+use std::{fs::File, process::exit};
 
 use clap::Parser;
+#[cfg(feature = "serde")]
+use lib_flutter_rust_bridge_codegen::dump;
 use lib_flutter_rust_bridge_codegen::{
     config_parse, frb_codegen_multi, get_symbols_if_no_duplicates, init_logger, RawOpts,
 };
 use log::{debug, error, info};
 
+const CONFIG_LOCATIONS: [&str; 3] = [".frb.yml", ".frb.yaml", ".frb.json"];
+
 fn main() -> anyhow::Result<()> {
     //  get valiable options from user input command
-    let raw_opts = RawOpts::parse();
+    let has_args = std::env::args_os().len() > 1;
+    let raw_opts = match RawOpts::try_parse() {
+        Ok(opts) => opts,
+        Err(err) if has_args => err.exit(),
+        // Try to parse a command file, if exists
+        Err(err) => 'from_file: {
+            for location in CONFIG_LOCATIONS {
+                let Ok(file) = File::open(location) else {continue};
+                break 'from_file serde_yaml::from_reader(file)
+                    .map_err(|err| anyhow::anyhow!("Could not parse {location}: {err}"))?;
+            }
+            _ = err.print();
+            eprintln!("Hint: To call flutter_rust_bridge_codegen with no arguments, fill in .frb.yml with your config.");
+            std::process::exit(1)
+        }
+    };
```

Comment:
uber-nit: maybe extract to a function, thus easier to read (eg no "break from_file"), also makes intent a little bit cleaner